tower/src/buffer/worker.rs

```diff
@@ -22,36 +20,38 @@ use tower_service::Service;
 /// implement (only call).
 #[pin_project]
 #[derive(Debug)]
-pub struct Worker<T, Request>
+pub struct Worker<S, Request, E2>
```

Comment:
Similarly, can this have a default so that I can still write `Worker<S, Request>` if I'm not overriding the error type?